.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+
+# Supported Python versions
+python:
+  - "3.6"
+
+services:
+  - postgresql
+
+
+# Install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install pytest
+  - pip install coveralls
+
+before_script:
+
+# Run tests
+script:
+  - coverage run --source=authors/ manage.py test 
+  - coverage report
+
+# Generate coverage report
+after_success:
+  - coveralls
```

Comment:
check on pep 8 style guide. At the end of the code, the last line should be blank

Reply:
on it.